ui/src/components/delete/QasDelete.stories.js

```diff
@@ -0,0 +1,72 @@
+import QasDelete from './QasDelete.vue'
+
+export default {
+  component: QasDelete,
+  title: 'Components/Delete',
+
+  parameters: {
+    docs: {
+      description: {
+        component: 'Insert line breaks where newlines `\n` occur in the string.'
+      }
+    }
+  },
+
+  argTypes: {
+    dialog: {
+      description: 'recebe as configurações do dialog'
+    },
+    customId: {
+      description: 'CustomID receives a customized id of the item to be deleted or if not the id to be used will be from the route'
```

Comment:
`Identifier to be deleted (if not sended the component will get id param of route).` não fica melhor?

Reply:
fica bem mais direto